templates/user/profile.html

```diff
@@ -79,6 +80,15 @@ <h1 name=username>{{ user.username or 'Insert a username' }}</h1>
   })
   localizeField.addEventListener('change', event => event.target.checked && localize())
 
+  const publishField = document.querySelector('[name=publish]')
```

Comment:
Well the issue you're on is supposed to solve [editing article's meta data](https://github.com/neomad-team/neomad.org/issues/27).
Changing the status of an article would be interested to be done in a list if a user needed to have a massive status change for multiple articles, but the purpose here is to be able to change non visible informations of an article (like the publication status, publication date, and later other infos such as the slug, transfer the owner and so on).
Hence the interest of having a status bar.

I'm running this version on my local copy to check the points of views ;)